src/js/common/components/Politician/PoliticianRetrieveController.jsx

```diff
@@ -28,7 +28,7 @@ class PoliticianRetrieveController extends Component {
       politicianWeVoteId,
     } = this.props;
     if (politicianWeVoteId !== previousPoliticianWeVoteId) {
-      // console.log('PoliticianRetrieveController componentDidUpdate politicianWeVoteId has changed');
+      console.log('PoliticianRetrieveController componentDidUpdate politicianWeVoteId has changed');
```

Comment:
I ask that console logs be commented out with checkins, so we can keep the production version console empty of most WeVote logging.